src/AllBuiltNodes.cpp

```diff
@@ -156,6 +155,7 @@ bool SaveAllBuiltNodes(Driver *self)
             case NodeBuildResult::kUpToDate:
             case NodeBuildResult::kRanSuccesfully:
             case NodeBuildResult::kRanSuccessButDependeesRequireFrontendRerun:
+            case NodeBuildResult::kUpToDataButDependeesRequireFrontendRerun:
```

Comment:
Should this be "up to date" instead of "up to data" ?

Reply:
fixed 8882e09